api/dimensions.go

```diff
@@ -55,13 +72,27 @@ func (api *DatasetAPI) getDimensions(w http.ResponseWriter, r *http.Request) {
 			return nil, err
 		}
 
+		totalCount := len(dimensions)
```

Comment:
It's better to instantiate variables close to where they are used.

In this case it is used once, so there isn't a need to instantiate a new variable but where you define `listOfDimensions := ...` setting the `TotalCount: len(dimensions)`